packages/app/components/token-picker/TokenPickerCommonTokens.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+
+import { RadioButton } from "@/ui";
+import { TokenIcon } from "../TokenIcon";
+import { Token } from "@/models/token/types";
+
+const COMMON_TOKENS = [
+  {
+    address: "0x532801ED6f82FFfD2DAB70A19fC2d7B2772C4f4b",
+    name: "Swapr on Gnosis chain",
+    symbol: "SWPR",
+    decimals: 18,
+    logoURI: "/assets/images/tokens/swapr.png"
+  },
+  {
+    address: "0x6A023CCd1ff6F2045C3309768eAd9E68F978f6e1",
+    name: "Wrapped Ether on Gnosis chain",
+    symbol: "WETH",
+    decimals: 18,
+    logoURI: "/assets/images/tokens/weth.png"
+  },
+  {
+    address: "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d",
+    name: "Wrapped XDAI",
+    symbol: "WXDAI",
+    decimals: 18,
+    logoURI: "/assets/images/tokens/wxdai.png"
+  },
+  {
+    address: "0x9C58BAcC331c9aa871AFD802DB6379a98e80CEdb",
+    name: "Gnosis Token on Gnosis chain",
+    symbol: "GNO",
+    decimals: 18,
+    logoURI: "/assets/images/tokens/gno.png"
+  }
+];
+
+export const TokenPickerCommonTokens = () => (
+  <div className="mt-5 px-6">
```

Comment:
Usually don't like to do this kind spacing inside of component, because questions like this arise:

- what if we want to use it in another place? 
- will the spacing be the same?

I don't think that is responsibility of `TokenPickerCommonTokens` to answer that. 

So I'd extract this spacing and add it on the place it's used.

```js
<div className="mt-5 px-6">
  <TokenPickerCommonTokens/>
</div>
```

Then if we need to use it in another place it will be easy as we define the framing on the place we use it and not inside.

Reply:
I have a deeper question. Do we need this as a new component? Can't it be declared inside `<Token Picker />` ? If we need it in another place, than we create a new component. I like the readability of creating the least amount of components possible.

Reply:
And we don't have new logic, so it's just a set of elements in the parent component.

Reply:
This thoughts can apply to TokenPickerList too.

Reply:
I think we can have it all on the `TokenPicker` and remove `TokenPickerList` and `TokenPickerCommonTokens`.